### PR TITLE
Added make commands to meshery-ui docs

### LINKED ISSUE #3636: [Docs]Update the make commands on meshery docs
#### Description
The new make commands have to be updated on https://docs.meshery.io/project/contributing-ui

#### Expected Behavior
All the make commands not present in the documentation already, need to be added. 

#### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker or Kubernetes

**Contributor Resources:**
- (Contributing Docs](https://docs.meshery.io/project/contributing)

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Added make commands to the meshery-ui docs
This PR fixes #3636 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 4d720544241ca0adf9196a145e77ff4df9cc8a6f at: https://60ffe25b3a48f4cc2d841e06--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-ui.md [hunk 1]
```diff
@@ -96,6 +96,27 @@ make build-ui
 
 > Changes are not recompiled directly, you will have to run to rebuild the UI to see them
 
+
+#### Build Meshery-UI
+
+```
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-ui.md:102]: This command is sans “meshery”, yeah?

File: docs/pages/project/contributing/contributing-ui.md [hunk 2]
```diff
@@ -96,6 +96,27 @@ make build-ui
...
+
+> While `make build-ui` builds all user interfaces on your local machine. 
+
+> `make build-meshery-ui` builds only meshery user interface on your local machine.
+
+
+#### Run Meshery
+To start running Meshery locally:
+```
+make run-fast
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-ui.md:114]: @utkarsh-pro should we displace the contents of `make run-local` with make run-fast and consolidate into one command?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/project/contributing/contributing-ui.md [hunk 3]
```diff
@@ -96,6 +96,27 @@ make build-ui
...
+```
+
+> Now, Meshery will run on the default port `http:localhost:9081`.
```
Comment (sudo-NithishKarthik) [inline docs/pages/project/contributing/contributing-ui.md:117]: `http://localhost:9081`

File: docs/pages/project/contributing/contributing-ui.md (hunk 1 above)
Comment (warunicorn19) [reply docs/pages/project/contributing/contributing-ui.md:102]: Yup

**Review by sudo-NithishKarthik — CHANGES_REQUESTED** [review]
We can also add the command for building provider ig 🤔

File: docs/pages/project/contributing/contributing-ui.md (hunk 3 above)
Comment (warunicorn19) [reply docs/pages/project/contributing/contributing-ui.md:117]: Oops, my bad.

**Comment by warunicorn19** [comment]
> We can also add the command for building provider ig 🤔

As in the `make build-provider-ui` ?

File: docs/pages/project/contributing/contributing-ui.md [hunk 4]
```diff
@@ -96,6 +96,27 @@ make build-ui
 
 > Changes are not recompiled directly, you will have to run to rebuild the UI to see them
 
+
+#### Build Meshery-UI
```
Comment (sudo-NithishKarthik) [inline docs/pages/project/contributing/contributing-ui.md:100]: I think we can keep this as a sub heading under `Build and export UI` to make it evident that it a part of building process.

**Comment by sudo-NithishKarthik** [comment]
> > We can also add the command for building provider ig 🤔
> 
> As in the `make build-provider-ui` ?
> @sudo-NithishKarthik any other command i need to add?

Yup. Infact, i don't see a section for contributing to provider ui itself, although it is not complicated, it should be there in the docs

No, i don't think there are any other commands to be added.

File: docs/pages/project/contributing/contributing-ui.md (hunk 2 above)
Comment (utkarsh-pro) [reply docs/pages/project/contributing/contributing-ui.md:114]: Yes, I think we should do that.

File: docs/pages/project/contributing/contributing-ui.md [hunk 5]
```diff
@@ -96,6 +96,27 @@ make build-ui
...
+
+```
+make build-meshery-ui
+```
+
+> While `make build-ui` builds all user interfaces on your local machine. 
+
+> `make build-meshery-ui` builds only meshery user interface on your local machine.
```
Comment (utkarsh-pro) [inline docs/pages/project/contributing/contributing-ui.md:108]: @warunicorn19 I don't think we have have to build the UI twice. `make build-ui` builds all of the UI at once so why build the UI again with `make build-meshery-ui`?

Comment (warunicorn19) [reply docs/pages/project/contributing/contributing-ui.md:108]: Makes sense, I will remove it.

File: docs/pages/project/contributing/contributing-ui.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/project/contributing/contributing-ui.md:114]: Sounds good, @utkarsh-pro 

@warunicorn19 will you do this:

Screen Shot 2021-07-27 at 8 15 28 AM

Comment (warunicorn19) [reply docs/pages/project/contributing/contributing-ui.md:114]: Yup I'll do it right away

Comment (warunicorn19) [reply docs/pages/project/contributing/contributing-ui.md:114]: @leecalcote just to be clear, i have to make an issue and reference this over there right?

**Comment by l5io** [comment]
🚀 Preview for commit b10fa04b46bcc3c611c45c0d8d4ae6c1e4c44c4d at: https://61014a0cf455f3dea5f2776d--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-ui.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/project/contributing/contributing-ui.md:103]: Right. 
FD168B5B-08D0-41E1-82D2-ECC57E1FBEDC

Comment (warunicorn19) [reply docs/pages/project/contributing/contributing-ui.md:103]: @leecalcote already did that #3656

Comment (leecalcote) [reply docs/pages/project/contributing/contributing-ui.md:103]: Oh! 🙂

File: docs/pages/project/contributing/contributing-ui.md (hunk 3 above)
Comment (leecalcote) [reply docs/pages/project/contributing/contributing-ui.md:106]:
```suggestion
> Now, Meshery will run on the default port `http://localhost:9081`.
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**